apps/server/Dockerfile

```
@@ -1,9 +1,45 @@
+
+# Switch to non-root user
+USER nodejs
+
+WORKDIR /app/apps/server
 
-EXPOSE 1284
+EXPOSE ${PORT}
 
-CMD ["bun", "index.js"]
+# Start the application with db push
+CMD bun --cwd /app/packages/db push && bun dist/index.js
```

Hadolint (2.12.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)